source/sass/sass-components/promo.scss

```
@@ -0,0 +1,27 @@
+.promo {
+  background-image: url("../img/index-background-2.jpg");
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  &__wrapper {
+    background-image: linear-gradient(to bottom, rgba(104, 183, 56, 0.85) 58%, transparent 58%)
+  }
```

1. Лучше использовать переменные. Тем более что уже определил
2. Более надёжно сделать псевдоэлемент нужный высоты, и "подкладывать его вниз. Тогда фон будет тянуться хорошо
```suggestion
.promo {
  position: relative;
  background-image: linear-gradient(to bottom, transparentize($green, 0.15) 58%, #ffffff 58%);

  &::before {
    background-image: url("../img/index-background-2.jpg");
    background-size: cover;
    background-repeat: no-repeat;
    background-position: center;
    content: "";
    height: 58%;
    width: 100%;
    position: absolute;
    top: 0;
    left: 0;
    z-index: -1;
  }
```